<launch>
	<node pkg="table_detection" type="viz_convex" name="viz_convex" output="screen">
	    <rosparam param="specify_index">[3]</rosparam>
		<param name="collection" value="final_table"/>
	</node>

</launch>
